.github/workflows/ci.yml

```
@@ -10,13 +10,15 @@ jobs:
       fail-fast: false
       max-parallel: 5
       matrix:
-        python-version: ['3.6', '3.7', '3.8', '3.9', '3.10']
-        django-version: ['2.2', '3.2']
+        python-version: ['3.8', '3.9', '3.10', '3.11', '3.12']
```

MUST: 3.12 は `-dev` が必要です。
https://github.com/actions/setup-python/blob/main/docs/advanced-usage.md#specifying-a-python-version

```suggestion
        python-version: ['3.8', '3.9', '3.10', '3.11', '3.12-dev']
```

最新は `3.12.0-rc.2` 
https://github.com/actions/python-versions/releases

---

https://github.com/beproud/django-newauth/pull/24 で `-dev` を指定してみたところ、テストが通りました。